### ISSUE TITLE
Need build script for Windows/SDL2

### ISSUE THREAD

**Issue by c0gent** [Nopey/rust-ocl-interop#2]
Directions are here:
https://github.com/AngryLawyer/rust-sdl2#windows-with-build-script

**Comment by Nopey** [comment]
SDL2 is only used by the tests, and so I think it is fine if the people running the tests have to install SDL2 and add it to their paths.
Perhaps the middle road would be to add code to the build script that automagically downloads and links to the latest SDL, but if that were a solution, wouldn't the `sdl2` crate be doing that?

**Comment by c0gent** [comment]
Yeah good point. I guess it depends how this library ends up being used (and if more than just a few of us end up using it).
